sql-plugin/src/main/311until340-all/scala/com/nvidia/spark/rapids/shims/GpuOptimizedCreateHiveTableAsSelectCommandShims.scala

```diff
@@ -1,5 +1,5 @@
 /*
```

Comment:
Why move this file?

Reply:
The type structure is different for Spark 3.4 and pre versions.
For Spark 311 until 340, `GpuCreateHiveTableAsSelectBase` is a subtype of `GpuDataWritingCommand`, but in Spark 3.4, `GpuCreateHiveTableAsSelectBase` is a subtype of `GpuRunnableCommand`